### DISCUSSION TITLE
Dark mode for client on Windows

### DISCUSSION THREAD

**Discussion by Dutchlikeme** [transmission/transmission#5457] [Q&A]
Is there a dark mode for the client on windows?
Or is there a way to customize the GUI?
I don't want a WEB UI, just want to change the client colors
![image](https://user-images.githubusercontent.com/100804523/234718521-04ff0a61-1d90-44be-ade0-f6e7a3734252.png)

**Comment by ngoclong19** [comment]
Please see https://github.com/transmission/transmission/issues/920.
